src/WebJobs.Script/Rpc/RpcInitializationService.cs

```diff
@@ -73,23 +78,23 @@ internal Task InitializeChannelsAsync()
             {
                 return Task.CompletedTask;
             }
-            if (_environment.IsLinuxContainerEnvironment())
-            {
-                return Task.CompletedTask;
-            }
             if (string.IsNullOrEmpty(workerRuntime) && _environment.IsPlaceholderModeEnabled())
             {
-                // Only warm up language workers in placeholder mode in worker runtime is not set
-                return Task.WhenAll(_languages.Select(runtime => _languageWorkerChannelManager.InitializeChannelAsync(runtime)));
+                // Only warm up language workers in placeholder mode if worker runtime is not set
+                if (_environment.IsLinuxContainerEnvironment())
```

Comment:
Both Linux consumption and AppServiceplan should start python worker at app lelvel